example/30-terraformer-apply-pod.yaml

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2017 SAP SE or an SAP affiliate company. All rights reserved.
```

Comment:
```suggestion
# Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved.
```

Reply:
I am not sure whether the year in the copyright needs to be maintained. At least in k/k it is not.

Reply:
Yep, it is not required to update the year, but my impression is that in this organization we use the current year for the newly created files.